submission-scripts/script-lsf-mpirun/magpie.lsf-mpirun

```diff
@@ -1869,6 +1869,20 @@ export ZEPPELIN_JOB="checkzeppelinup"
 #
 # export ZEPPELIN_NOTEBOOK_DIR="/lustre/${USER}/zeppelin/notebook"
 
+# Zeppelin Notebook Authorization
+#
+# This will enable username, password, and roles to be established.
```

Comment:
just one tweak, since this is now required, perhaps the language should be more on point about it.  Perhaps "Set username, password, role ... this must be set before running Zeppelin".   And then also add this setting to the "Setup the essentials for Zeppelin." part of README.zeppelin.

Reply:
Ah good catch! Working it